lib/rides/models/ride.dart

```diff
@@ -203,6 +203,10 @@ extension RideStatusExtension on RideStatus {
     return this == RideStatus.approved;
   }
 
+  bool isRealRider() {
```

Comment:
can we add a comment, what this method checks, i think the name is kinda weird

Reply:
Yes I'm not happy with the name either... Any ideas?

Reply:
no that's the problem xD, maybe just a short comment or something.

Reply:
maybe "canSeeFullRideInformation" ?

Reply:
yes that's good then it's clear why we need the method.